docker/workspace-base-dev/Dockerfile

```diff
@@ -24,3 +25,18 @@ RUN apt-get -y update \
 # Activate some bash completion
 RUN echo "source /etc/bash_completion" >> ~/.bashrc \
  && echo "source /usr/share/colcon_argcomplete/hook/colcon-argcomplete.bash" >> ~/.bashrc
+
+# Install opengl
+RUN apt-get update \
+  && apt-get install -y -qq --no-install-recommends \
+    libglvnd0 \
+    libgl1 \
+    libglx0 \
+    libegl1 \
+    libxext6 \
+    libx11-6 \
+  && rm -rf /var/lib/apt/lists/*
+
+# Env vars for the nvidia-container-runtime.
+ENV NVIDIA_VISIBLE_DEVICES all
+ENV NVIDIA_DRIVER_CAPABILITIES graphics,utility,compute
```

Comment:
I'm curious to learn what these are for. This is possibly the correct location (though merged with the apt-install earlier in this file). Is this required for the trimesh python package stack?

Reply:
Yeah, this is what I had to do to get OpenGL working in the container. The trimesh scene view uses that to show the meshes.

Reply:
Although I don't think it's required if we don't use the scene viewer. https://trimsh.org/trimesh.viewer.html

Reply:
Either way, I believe this belongs in the other Dockerfile since it has to do with the running of the ROS workspace, but development tools.